micromamba/tests/test_shell.py

```diff
@@ -86,7 +86,7 @@ def test_hook(self, shell_type):
         elif shell_type == "cmd.exe":
             assert res == ""
         elif shell_type == "tcsh":
-            assert res.count(mamba_exe) == 2
+            assert res.count(mamba_exe) == 3
```

Comment:
Laybe this case could be put wiht "zsh", "bash" and "posix" above? Also, what does this return code mean?